samples/nrf9160/http_update/full_modem_update/README.rst

```diff
@@ -34,17 +37,29 @@ The sample supports the following development kit, version 0.14.0 or higher:
 
 .. include:: /includes/spm.txt
 
-
 Building and running
 ********************
 
-.. |sample path| replace:: :file:`samples/nrf9160/application_update/full_modem_update`
+.. |sample path| replace:: :file:`samples/nrf9160/http_update/full_modem_update`
 
 .. include:: /includes/build_and_run.txt
 
 The sample is built as a non-secure firmware image for the ``nrf9160dk_nrf9160ns`` build target.
 Because of this, it automatically includes the :ref:`secure_partition_manager`.
 
+You can customize the firmware files downloaded by the sample through the following Kconfig options in the :file:`prj.conf` file:
+
+* `CONFIG_DOWNLOAD_HOST` - it sets the hostname of the server where the updates are located.
```

Comment:
We might consider putting this information in a table

Reply:
I actually just opted for using the option:: directive